vagrant/openshift/Vagrantfile

```diff
@@ -251,23 +257,23 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Landrush is used together with wildcard dns entries to map all
   # routes to the proper services
   if $windows
-    config.hostmanager.enabled = true
-    config.hostmanager.manage_host = true
+    config.hostmanager.enabled = $vmSimDNS
+    config.hostmanager.manage_host = $vmSimDNS
     config.hostmanager.ignore_private_ip = false
     config.hostmanager.include_offline = true
 
     config.hostmanager.aliases = %w(fabric8.vagrant.f8 jenkins.vagrant.f8 gogs.vagrant.f8 nexus.vagrant.f8 hubot-web-hook.vagrant.f8 letschat.vagrant.f8 kibana.vagrant.f8 taiga.vagrant.f8 fabric8-forge.vagrant.f8)
   else
-    config.landrush.enabled = true
+    config.landrush.enabled = $vmSimDNS
     config.landrush.tld = $tld
     config.landrush.guest_redirect_dns = false
-    config.landrush.host_ip_address = '172.28.128.4'
+    config.landrush.host_ip_address = $vmIP
   end
 
   config.vm.box = "jimmidyson/centos-7.1"
   config.vm.box_version = "= 1.3.1"
 
-  config.vm.network "private_network", ip: "172.28.128.4"
+  config.vm.network "private_network", ip: $vmIP, macaddress: $vmMAC
```

Comment:
I ran some tests against virtualbox.

The reason for this change is that the network scripts in the box have been hard coded to use a MAC address. To get network connectivity under KVM, I needed to set the address explicitly. Presumably virtualbox is doing this by default in some way. 

Terry

> On 6 Apr 2016, at 19:12, Jimmi Dyson <notifications@github.com> wrote:
> 
> In vagrant/openshift/Vagrantfile:
> 
> >    end
> >  
> >    config.vm.box = "jimmidyson/centos-7.1"
> >    config.vm.box_version = "= 1.3.1"
> >  
> > -  config.vm.network "private_network", ip: "172.28.128.4"
> > +  config.vm.network "private_network", ip: $vmIP, macaddress: $vmMAC
> Has seeing the mac address been tested on both virtualbox & libvirt?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>